yt_dlp/utils.py

```diff
@@ -5407,3 +5407,7 @@ def _cancel_all_tasks(loop):
 
 
 has_websockets = bool(compat_websockets)
+
+
+def get_first(obj, keys, **kwargs):
+    return traverse_obj(obj, (..., *variadic(keys)), **kwargs, get_all=False)
```

Comment:
This was made a function in youtube.py because it was being used a lot (33 hits). Considering it is just thin wrapper around travsere_obj, idk if this is useful enough in general to be a util function. Excluding this extractor (6 hits), this pattern is only used 5 other times

```py
yt_dlp • extractor\facebook.py:
  408:             title = traverse_obj(media, (..., 'title', 'text'), get_all=False)
  409              description = traverse_obj(media, (
  410                  ..., 'creation_story', 'comet_sections', 'message', 'story', 'message', 'text'), get_all=False)
  411:             uploader_data = (traverse_obj(media, (..., 'owner'), get_all=False)
  412:                              or traverse_obj(post, (..., 'node', 'actors', ...), get_all=False) or {})

yt_dlp • extractor\tiktok.py:
  820:             traverse_obj(render_data, (..., 'aweme', 'detail'), get_all=False), url)
```

Reply:
Yeah, I think I'll go back to using traverse_obj